src/App.tsx

```diff
@@ -15,71 +16,76 @@ export const goods = [
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    const { selectedGood } = this.state;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        { !selectedGood
+          ? <h1 className="title">No goods selected</h1>
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              {selectedGood}
+              {' '}
+              is selected
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.setState({ selectedGood: '' });
+                }}
+              />
+            </h1>
+          )}
+        <table className="table">
+          <tbody>
+            {goods.map(food => {
+              const booleanFood = selectedGood === food;
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+              return (
+                <tr
+                  data-cy="Good"
+                  className={
+                    classNames(
+                      { 'has-background-success-light': booleanFood },
+                    )
+                  }
+                >
+                  <td>
+                    <button
+                      data-cy="AddButton"
+                      type="button"
+                      className={
+                        classNames('button', { 'is-info': booleanFood })
+                      }
+                      onClick={() => {
+                        this.setState({ selectedGood: food });
+                      }}
+                    >
+                      { booleanFood ? '-' : '+' }
```

Comment:
you show different text (`+`/`-`) in button but don't change the function
in case, when `-` is displayed you should deselect it (set `''` or better `null`)

Reply:
Добрый вечер. Честно говоря не понял суть. Если возможно написать чуть развернутее. И если можно на украинском языке.

Reply:
привіт, якщо у тебе виникло питання - пиши його, будь ласка, в чат в слек, і, лише коли все виправиш, тоді ре-ріквестай ревью, на всі запитання ми відповідаєм у чат, а ревью ріквестаєм лише на завершену/виправлену роботу.
Якщо перевести коментар Тараса у транслейтері, буде зрозуміло, що він каже: ти показуєш різний текст (+/-)  у кнопці, проте не змінюєш функцію.

Натискання на кнопку - має деселекнути твій вибір, тобто зробити те саме, що при кліку на іконку хрестика